python_cmake_module/cmake/Modules/FindPythonExtra.cmake

```diff
@@ -211,7 +222,11 @@ if(PYTHONINTERP_FOUND)
   endif()
 
   if(WIN32)
-    set(PythonExtra_EXTENSION_EXTENSION ".pyd")
+    if("${CMAKE_BUILD_TYPE} " STREQUAL "Debug ")
+      set(PythonExtra_EXTENSION_EXTENSION "_d.pyd")
```

Comment:
If this can be detected at CMake time wouldn't it be better to print a clear error message then let it fail later with a not that clear error?